### PR TITLE
Update Paid Editions Blurb

### PR DESCRIPTION
**Pull request by jobannon**
# Summmary 
- This updates the paid editions Blurb
- As usual, open to feedback; alt presentations
- Linked Style PR https://github.com/FusionAuth/fusionauth-site/pull/722/files

# CSS updated
The updated CSS is 
in `website-docs.scss`

```
      &.note {
        border-left: 4px solid $blue-medium;

        &.paid {
          border-left: 4px solid $orange-light;

          .icon.note::before {
            color: $orange-medium;
            content: '\f7ba';
            font-family: 'Font Awesome 5 Pro';
            font-weight: 300;
          }
        }
```

# See previous conversations

- https://github.com/FusionAuth/fusionauth-site/pull/661/files#r631421551
    - Not entirely sure how to address smooshed callouts... per @mooreds we could keep "stacked callouts" separated but I also wonder if there was a way that they could look a bit more cohesive if they are stacked (via visual layout/css/page planning).  
- https://github.com/FusionAuth/fusionauth-site/pull/661

# Screenshot
Screen Shot 2021-05-26 at 9 32 57 PM
for example at https://site-local.fusionauth.io/docs/v1/tech/account-management/updating-user-data/

### CODE REVIEW

**Comment by mooreds** [comment]
Like it a lot.

The only feedback I have is that we have two paid callouts, one for API docs and one everywhere else. This takes care of the latter, but wasn't sure if you wanted to address the former?

https://github.com/FusionAuth/fusionauth-site/blob/master/site/docs/v1/tech/shared/_premium-edition-blurb-api.adoc

I wouldn't worry to much about an automatic solution to stacked callouts. That can be addressed on a case by case basis, which I often do by adding some text between the callouts I add.

**Comment by jobannon** [comment]
image

What are your thoughts on this?

Updated css would be 

```      
> span.paid {
        border-top: solid 1px;
        border-bottom: solid 1px;
        color: $gray-5;
        font-size: 10px;
        font-weight: 500;
        text-transform: uppercase;
      }
```
under `section.docs > article > p >`

I might need a refresher on releasing this if it looks good.

**Review by mooreds — APPROVED** [review]
Other than one small comment, looks good.

File: site/docs/v1/tech/shared/_premium-edition-blurb-api.adoc [hunk 1]
```diff
@@ -1 +1 @@
-A paid edition of FusionAuth is required to utilize {premium_feature}.
+icon:radiation-alt[type=fal] [paid]#A paid edition of FusionAuth is required to utilize {premium_feature}.# icon:radiation-alt[type=fal]
```
Comment (robotdan) [inline site/docs/v1/tech/shared/_premium-edition-blurb-api.adoc:1]: See note from https://github.com/FusionAuth/fusionauth-style/pull/15#issuecomment-856311918 

If we are going to use this to indicate a paid feature, I would rather use our actual SVG for reactor to match the brochure site. We also use this logo in the admin UI. 

image

**Pull request closed without merging**